WalletWasabi.Fluent/Views/MainWindow.axaml

```diff
@@ -19,7 +19,7 @@
         ExtendClientAreaToDecorationsHint="True"
         ExtendClientAreaChromeHints="Default, PreferSystemChrome, OSXThickTitleBar"
         Title="Wasabi Wallet"
-        WindowState="{Binding WindowState, Mode=TwoWay}"
```

Comment:
I think this breaks the shutdown prevention, where we apply the window state again from the config if it is minimized just to bring the window in the front and display the shutdown prevention dialog.